modules/pagereader.py

```diff
@@ -50,6 +50,46 @@ def readPage(site, extension=False):
         raise("There has been a connection error after three attempts.")
 
 
+def read_page(site, extension=False):
```

Comment:
This should be `read_first_page` right?

Reply:
No. It's correct. The only problem is that there is a bit of repeated code.

Reply:
Ok